packages/lesswrong/components/review/AnnualReviewPage.tsx

```diff
@@ -138,6 +138,7 @@ export const AnnualReviewPage = ({classes}: {
 
     if (value === 'reviewVoting') {
       newPathname = `/reviewVoting/${reviewYear}`;
+      newQuery = { }
```

Comment:
what dis?

Reply:
That results the query to empty (which you want, because otherwise some crud accumulates)

Reply:
i see it kills the query